src/main/java/org/example/dao/CityDao.java

```diff
@@ -0,0 +1,97 @@
+package org.example.dao;
+
+import org.example.entity.City;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CityDao {
```

Comment:
you can create AbstractDao where will be located basic logic which is common for all Dao's.